UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -564,19 +567,22 @@ ParsegraphicNode (
 
   @param[in]  Fdt               Address of the Fdt data.
   @param[in]  SubNode           first Sub node of the PCI root bridge node.
+  @param[in]  AddressCells      #address-cells for serial port node 'reg' property.
+  @param[in]  SizeCells         #size-cells for serial port node 'reg' property.
 **/
 VOID

```

Comment:
Could you help to create another function for the independent  Serial Port support?
With this change in current platform , it will assert in line 610.

Reply:
Thanks for the comment. you are right, "reg-shift" is optional from spec so assert should be removed. I will update this PR.

Reply:
It was updated.